packages/ui-table/src/Table/README.md

```diff
@@ -167,141 +351,412 @@ class Example extends React.Component {
+        })
+      } else {
+        this.setState({
+          sortBy: id,
+          ascending: true
+        })
+      }
+    }
+
+    handleColTextAlignChange(id, value) {
+      this.setState((state) => ({
+        colTextAligns: {
+          ...state.colTextAligns,
+          [id]: value
+        }
+      }))
+    }
+
+    renderHeaderRow(direction) {
+      const { headers } = this.props
+      const { colTextAligns, sortBy } = this.state
+
+      return (
+        <Table.Row>
+          {(headers || []).map(({ id, text, width }) => (
+            <Table.ColHeader
+              key={id}
+              id={id}
+              width={width}
+              {...(direction && {
+                textAlign: colTextAligns[id],
+                stackedSortByLabel: text,
+                onRequestSort: this.handleSort,
+                sortDirection: id === sortBy ? direction : 'none'
+              })}
+            >
+              {text}
+            </Table.ColHeader>
+          ))}
+        </Table.Row>
+      )
+    }
+
+    renderOptions() {
+      const { headers } = this.props
+      const { colTextAligns } = this.state
+
+      return (
+        <ToggleGroup
+          size="small"
+          toggleLabel="Set text-align for columns"
+          summary="Set text-align for columns"
+          background="default"
+        >
+          <Table caption="Set text-align for columns">
+            <Table.Head>{this.renderHeaderRow()}</Table.Head>
+            <Table.Body>
+              <Table.Row>
+                {Object.entries(colTextAligns).map(([headerId, textAlign]) => {
+                  return (
+                    <Table.Cell key={headerId}>
+                      <RadioInputGroup
+                        description={
+                          <ScreenReaderContent>
+                            Set text-align for column: {headerId}
+                          </ScreenReaderContent>
+                        }
+                        name={`columnTextAlign_${headerId}`}
+                        value={textAlign}
+                        margin="0 0 small"
+                        size="small"
+                        onChange={(e, value) =>
+                          this.handleColTextAlignChange(headerId, value)
+                        }
+                      >
+                        <RadioInput label="start" value="start" />
+                        <RadioInput label="center" value="center" />
+                        <RadioInput label="end" value="end" />
+                      </RadioInputGroup>
+                    </Table.Cell>
+                  )
+                })}
+              </Table.Row>
+            </Table.Body>
+          </Table>
+        </ToggleGroup>
+      )
+
+      return (
+        <FormField id="columnTextAlign" label="Set column text-align">
+          <Flex margin="0 0 medium">
+            {Object.entries(colTextAligns).map(([headerId, textAlign]) => {
+              return (
```

Comment:
This whole Formfield is missing from the function-based component.

Reply:
I just realized that the `renderOptions` function in the class-based example has two return statements, with the second one including the FormField. I think we should remove the unreachable return statement completely from the class-based component.

Reply:
I agree, it should be removed

Reply:
Done